### DISCUSSION TITLE
Error Running a new serenity app

### DISCUSSION THREAD

**Discussion by gfo2007** [serenity-is/Serenity#5845] [Help] [answered]
Hi,
I just installed serene and updated the nuget packages. Whenever I try to run the application, I get the error in the image attached. 

Pleased help
![serenity core error](https://user-images.githubusercontent.com/14317420/124435827-e564f900-dd64-11eb-99f5-2c75b18cd2f3.PNG)

**Comment by VictorTomaili** [comment, marked as answer]
Issue related to `StackExchange.Exceptional` and `System.Data.SqlClient`
If you don't need it, just remove `StackExchange.Exceptional`. 
Serenity was not tested with the latest packages. If you didn't have strict requirements for updates, please wait for a new release.

**Reply by gfo2007** [reply]
@VictorTomaili thanks for the quick reply. That error is gone. My problem I am facing right now is that the Login Page is not loading. 

Please the image is below
![asp net core error](https://user-images.githubusercontent.com/14317420/124463590-a3e34680-dd82-11eb-9457-a6dc6026f449.PNG)

**Reply by VictorTomaili** [reply]
`dotnet sergen restore` then rebuild.

**Reply by Surenthiran** [reply]
am also getting same blank screen after updating packages to 5.1.43.
if i run `dotnet sergen restore` 

Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET program, but dotnet-sergen does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.

**Reply by VictorTomaili** [reply]
https://www.nuget.org/packages/sergen/

**Reply by Surenthiran** [reply]
after running dotnet sergen restore

PS C:\Users\Administrator\source\repos\CRM\CRM\CRM.Web> dotnet sergen restore
Processing: Hangfire.Core
Processing: Hangfire.SqlServer
Processing: ResponsiveFileManager.AspNetCore
Processing: Serenity.Assets
Processing: Serenity.Scripts
Processing: Serenity.Extensions

still geting 
![image](https://user-images.githubusercontent.com/2942248/124732120-8a7ffd00-df30-11eb-91fc-c2bfab6eb401.png)

console error as below 

![image](https://user-images.githubusercontent.com/2942248/124732267-aedbd980-df30-11eb-91e3-955d01ae1a4e.png)

**Reply by VictorTomaili** [reply]
clean and rebuild, check the typescript version and update if required. If your app still doesn't run well, please create a new sample and update just serenity packages instead of all. As I said, serene won't be tested with the latest third-party packages. But this type of error is related to missing javascript files or when typescript is not compiled well.

**Reply by Surenthiran** [reply]
cleaned the code updated typescript to 4.3.2

after updating am getting following error 

![image](https://user-images.githubusercontent.com/2942248/124759885-c70e2180-df4d-11eb-99af-ec52d9c0b30b.png)

**Reply by VictorTomaili** [reply]
I recheck the required version of the typescript. Update for this version. And create a new project then test it. Check for diffs etc. 
https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Core/Serene.Core.csproj#L33

**Comment by gfo2007** [comment]
@VictorTomaili thanks very much for your responses. The _dotnet sergen restore_ command worked like a charm.

@Surenthiran can I close the issue?

Regards
